src/elements/content-picker/__tests__/Footer.test.js

```diff
@@ -25,6 +25,19 @@ describe('elements/content-picker/Footer', () => {
             expect(wrapper.find('.footer-child').length).toBe(1);
         });
 
+        test('should render footer with disabled button', () => {
+            // Wrapper with two nodes
```

Comment:
this comment isn't really necessary and could become outdated if buttons are added or removed

Reply:
yes you're right, these are more or less leftover from me explaining it to myself as i went along